exercises/darts/darts.py

```diff
@@ -0,0 +1,2 @@
+def Score(x, y):
```

Comment:
Method names should start with lowercase.

```suggestion
def score(x, y):
```

Reply:
Changed to match score function.